### PR TITLE
[Visual Testing] Update comment job in workflow to use custom script

### PR DESCRIPTION
**Pull request by KshitijThareja**
## Description
Updates the comment job in the visual testing workflow to use `github-script` action instead of using a third-party action.

## Changelog

- [#PR no]
  - **Description:** Updates the comment job in the visual testing workflow
  - **Products impact:** -
  - **Addresses:** -
  - **Components:** -
  - **Breaking:** -
  - **Impacts a11y:** -
  - **Guidance:** -

[#PR no]: PR link

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change is described in the changelog section above

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

File: .github/workflows/visual_tests.yml [hunk 1]
```diff
@@ -64,22 +64,50 @@ jobs:
...
+        
+      - name: Create or update comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs');
+            const commentBody = fs.readFileSync('comment_body.md', 'utf8');
+            
+            const { data: comments } = await github.rest.issues.listComments({
```
Comment (AlexVelezLl) [inline .github/workflows/visual_tests.yml]: It seems this listComments returns paginated data up to 30 results https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28. In Kolibri we use [this function](https://github.com/learningequality/kolibri/blob/5582278a1d4179bc2191e9545275a4d93b647bf1/.github/githubUtils.js#L116) to find the generated comment, and then in the workflow we just import the file, and return the [function result](https://github.com/learningequality/kolibri/blob/5582278a1d4179bc2191e9545275a4d93b647bf1/.github/workflows/pr_build_comment.yml#L38). I think we can use a similar strategy moving this code to a file `.github/githubUtils.js` here in KDS to have the benefits of linting and code formatting.

Comment (KshitijThareja) [reply .github/workflows/visual_tests.yml]: Sure. I'll change the current setup to something similar to what is used in Kolibri then 👍

**Comment by KshitijThareja** [comment]
Hello @AlexVelezLl! I've made the required changes to the comment job and shifted the find comment and generate comment part to githubUtils.js taking reference from the Kolibri implementation.

**Review by AlexVelezLl — APPROVED** [review]
Thank you @KshitijThareja!! :tada:. Looks good to me! Lets merge this, and test how it works with the KButton visual tests PR.

**Pull request merged**